util/binwarp/user-facing.lisp

```diff
@@ -0,0 +1,97 @@
+;;;; user-facing.lisp
+
+(in-package :stumpwm)
+
+;;; This file exists because I needed a good and obvious way
+;;; for user to define their own binwarp-mode settings while being in the
+;;; :STUMPWM package. If there's a better way, I'll be glad to hear about it!
```

Comment:
Before I can suggest a better™ way I need to understand why you need this to be in the stumpwm package.

Reply:
Sure. The reason for that was that the symbol for `kbd` is not present in `binwarp` package and, when I was uquote-splicing (`,@`) the list of keybindings, `kbd` was automatically interned to `binwarp`. Because it was unbing in `binwarp`, it was causing errors during macroexpansion.

The only solution I came up with was to `defvar` the `*default-binwarp-keymap*` in the `:stumpwm` package and refer to it in `define-binwarp-mode` in the same package.

So, if there is a way to make `kbd` in the `*default-binwarp-keymap*` to be interpreted as `stumpwm:kbd` and not `binwarp:kbd`, then I won't need to pollute `:stumpwm`.